Tools/serial/generate_config.py

```diff
@@ -26,41 +26,93 @@
 # with QGC (parameter metadata)
 serial_ports = {
     # index 0 means disabled
+
+    # Generic
+#     "URT0": {
+#         "label": "UART 0",
+#         "index": 1,
+#         "default_baudrate": 57600,
+#         },
+#     "URT1": {
+#         "label": "UART 1",
+#         "index": 2,
```

Comment:
While we still have a chance to change it: should we be consistent and use URT1 -> index 1 and remove URT0? Or shift the range?

Reply:
We could use `TTYx` instead of `URTx`. I think that's the most consistent. If you agree I'll simply shift the ranges.

Reply:
For Omnibus I think we should go for USART6, because that's what the label says. The NuttX ttySX are not fixed. For example we have one of the UARTs configured as I2C currently on Omnibus, but this could change in future.
So I somewhat prefer URTx (with URT0).

Reply:
Ok, let's give that a try and see how it goes. If the idea is to treat it like a user facing label where do you see the need for URT0?

Reply:
I don't, but wasn't the idea to think ahead and keep options open?